### PR TITLE
Removed Vidhance header.

### PR DESCRIPTION
**Pull request by emilwestergren**
Vidhance header is no longer part of this repo.

### CODE REVIEW

**Comment by emilwestergren** [comment]
@sebastianbaginski peer review

File: vidhance/vidhance.h [hunk 1]
```diff
@@ -1,339 +0,0 @@
...
-typedef void* vidhance_motion_sensor_angularVelocityMeasurements;
-typedef void* vidhance_contrast_settings;
-
-/* Settings */
-extern vidhance_settings vidhance_settings_new();
-extern void vidhance_settings_free(vidhance_settings settings);
-
-/* Motion Settings */
-extern vidhance_motion_settings vidhance_settings_getMotion(const vidhance_settings settings);
-extern void vidhance_motion_settings_setComplexity(vidhance_motion_settings settings, int complexity);
```
Comment (sebastianbaginski) [inline vidhance/vidhance.h]: `complexity` can be `const`.

**Comment by sebastianbaginski** [comment]
just kidding, looks ok

**Comment by emilwestergren** [comment]
Haha :)

**Pull request merged**